DM: add dm 5.4.0 performance test report (#7466)
pingcap/docs: dm/dm-benchmark 5.4.0.md

> ---

Reviewer: Please pay attention to the file name. It should be dm-benchmark-v5.4.0.md.

title: DM 5.4.0 Benchmark Report

> summary: Learn about the performance of 5.3.0.

Reviewer: ```suggestion
summary: Learn about the performance of 5.4.0.
```

---

# DM 5.4.0 Benchmark Report

> The performance of TiDB Data Migration (DM) v5.4.0 is nearly the same as that of TiDB DM v5.3.0. Therefore, you can refer to [DM 5.3.0 Benchmark Report](/dm/dm-benchmark-v5.3.0.md) to learn about the test purpose, environment, scenarios, and results of DM v5.3.0.

Reviewer: ```suggestion
The benchmark testing data of TiDB Data Migration (DM) v5.4.0 is nearly the same as that of DM v5.3.0. To learn the test purpose, environment, scenarios, and results of DM v5.3.0, refer to [DM 5.3.0 Benchmark Report](/dm/dm-benchmark-v5.3.0.md). 
```
